### PR TITLE
Update SDWebImage to the latest version

### PR DESCRIPTION
**Pull request by leoz**

### CODE REVIEW

File: RNFastImage.podspec [hunk 1]
```diff
@@ -16,6 +16,6 @@ Pod::Spec.new do |s|
   s.source_files  = "ios/**/*.{h,m}"
 
   s.dependency 'React-Core'
-  s.dependency 'SDWebImage', '~> 5.8'
+  s.dependency 'SDWebImage', '~> 5.9.4'
```
Comment (MiguelGrability) [inline RNFastImage.podspec:19]:
```suggestion
  s.dependency 'SDWebImage', '~> 5.10.0'
```

File: RNFastImage.podspec [hunk 2]
```diff
@@ -16,6 +16,6 @@ Pod::Spec.new do |s|
...
   s.dependency 'SDWebImageWebPCoder', '~> 0.6.1'
```
Comment (MiguelGrability) [inline RNFastImage.podspec:20]:
```suggestion
  s.dependency 'SDWebImageWebPCoder', '~> 0.7.0'
```

**Review by MiguelGrability — COMMENTED** [review]
I think that is a good opportunity to take this PR and use the latest version of both dependencies

**Comment by leoz** [comment]
@DylanVann , @MiguelGrability , is this good to merge?

**Comment by leoz** [comment]
Please, merge my pull request.

**Comment by pribeh** [comment]
Tested. Works. Please merge.

**Comment by evanjmg** [comment]
Please merge @DylanVann

**Comment by leoz** [comment]
@DylanVann, @AngelTrinidad, @taschik, could you please accept this change?

**Review by DylanVann — APPROVED** [review]
LGTM!

**Pull request merged**